mm/2s2h/Rando/CheckTracker/CheckTracker.cpp

```diff
@@ -189,103 +227,182 @@ void Window::DrawElement() {
                         auto& randoSaveCheck = RANDO_SAVE_CHECKS[checkId];
                         ImGui::PushStyleColor(ImGuiCol_Text, randoSaveCheck.obtained ? UIWidgets::Colors::LightGreen
                                                                                      : UIWidgets::Colors::White);
-                        if (checkTrackerShouldShowRow(randoSaveCheck.obtained)) {
+                        if (checkTrackerShouldShowRow(randoSaveCheck.obtained, randoSaveCheck.skipped)) {
+                            ImGui::BeginGroup();
+                            float cursorPosY = ImGui::GetCursorPosY();
+                            if (Rando::StaticData::Checks[checkId].randoCheckType == RCTYPE_OWL) {
+                                ImGui::SetCursorPosY(ImGui::GetCursorPosY() + 3.5f);
+                            }
+                            ImGui::Image(setCheckTypeTextureProperties(checkId), imageSize, ImVec2(0, 0), ImVec2(1, 1),
+                                         colorFilter);
```

Comment:
Instead of calling `setCheckTypeTextureProperties`, which returns the textureId and sets some static file level variables, just make a func called `DrawIcon(RandoCheckId randoCheckId)` that does this inside.
```suggestion
                            DrawIcon(checkId);
```